hd/etc/perso_module/fratrie.txt

```diff
@@ -134,13 +134,13 @@
 %if;(op_m=1 and (half_father != 0 or half_mother != 0))
   <h2>[*half siblings]</h2>
   %if;(half_father != 0)
-        [*on %s's side:::] %apply;short_display_person("father")
+      %apply;on_side%with;%apply;very_short_display_person("father")%end;
       %if;(father.nb_families > 1 and half_father != 0)
         %foreach;father.family;
           %if;(spouse.index!=mother.index and half_father!=0 and spouse.has_children)
             <ul>
               %apply;li_SD("spouse")
-                [with] %apply;short_display_person("spouse")
+                [with] %apply;very_short_display_person("spouse")
```

Comment:
Yes it is. In english, the text "on xxx's side" breaks down if you have dates and titles attached to the name.